### PR TITLE
Feat 409: Shows a warning if a user is logged in when registering

### PR DESCRIPTION
**Pull request by Henriki2305**
## WHY:

Fixes [#409 ]

### What has been changed (if possible, add screenshots, gifs, etc. )

Adds a warning that shows if a user is currently logged in

image

## Check off each of the following tasks as they are completed

- [x] I have reviewed the changes myself in this PR. Please check the [self-review document](https://github.com/OpenVAA/voting-advice-application/blob/main/docs/contributing/self-review.md)
- [ ] I have added or edited unit tests.
- [x] I have run the unit tests successfully.
- [ ] I have run the e2e tests successfully.
- [x] I have tested this change on my own device.
- [x] I have tested this change on other devices (Using Browserstack is recommended).
- [x] I have tested my changes using the [WAVE extension](https://wave.webaim.org/extension/)
- [x] I have added documentation where necessary.
- [x] Is there an existing issue linked to this PR? #409 

**Clean up your git commit history before submitting the pull request!**

### CODE REVIEW

**Review by samuliys — CHANGES_REQUESTED** [review]
The user usually registers using the link, which skips this page completely. Therefore, I think that this warning should also be added to the page where the candidate sets their password for the first time.

In addition, the warning states that the user should log out, but there is no logout button visible on the page. Should it be visible?

File: frontend/src/lib/templates/frontPage/FrontPage.svelte [hunk 1]
```diff
@@ -48,6 +48,7 @@ is based on.
     <h1>{$page.data.election.name}</h1>
   </svelte:fragment>
 
+
```
Comment (joonatanaatos) [inline frontend/src/lib/templates/frontPage/FrontPage.svelte:51]: Unnecessary newlines shoud be eschewed

File: frontend/src/lib/candidate/components/PasswordField/PasswordField.svelte [hunk 2]
```diff
@@ -4,6 +4,7 @@
   export let password = '';
   export let autocomplete = '';
   export let id = 'password';
+  $: fieldId = id;
```
Comment (joonatanaatos) [inline frontend/src/lib/candidate/components/PasswordField/PasswordField.svelte:7]: What is the purpose of this? I'd assume Svelte automatically re-renders when the props change. If there is a good reason, an inline comment should be added

**Review by samuliys — APPROVED** [review]
LGTM after addressing the `fieldId` situation

**Pull request merged**